tests/cases/standalone/common/flow/flow_advance_ttl.result

```diff
@@ -8,6 +8,20 @@ CREATE TABLE distinct_basic (
 
 Affected Rows: 0
 
+-- should fail
+-- SQLNESS REPLACE id=\d+ id=REDACTED
+CREATE FLOW test_distinct_basic SINK TO out_distinct_basic AS
+SELECT
+    DISTINCT number as dis
+FROM
+    distinct_basic;
+
+Error: 3001(EngineExecuteQuery), Invalid query: Source table `greptime.public.distinct_basic`(id=REDACTED) has instant TTL, flow will only evaluate to empty results with such table, use a small ttl instead of instant
```

Comment:
sugg: "Instant TTL is not supported under batching mode. Consider using a TTL longer than flush interval".

And the error type could be unsupported